test/tarball/verifySemVer.spec.js

```diff
@@ -0,0 +1,61 @@
+// Require Node.js Dependencies
+import { describe, test } from "node:test";
+import assert from "node:assert";
+import { getSemVerWarning } from "../../src/utils/index.js";
+
+// CONSTANTS
+const KVersionsToWarn = new Set(["0.0.0", "0.0", "0"]);
+
+const [firstVersion, secondVersion, lastVersion] = KVersionsToWarn.values();
+
+describe("getSemVerWarning", () => {
+  test("should return a warning for version '0.0.0'.", () => {
+    assert.equal(KVersionsToWarn.has(firstVersion), true);
+
+    const expectedWarning = {
+      skind: "invalid-semver",
+      file: "package.json",
+      value: firstVersion,
+      location: null,
+      i18n: "sast_warnings.invalidSemVer",
+      severity: "Information",
+      experimental: false
+    };
+
+    assert.deepEqual(getSemVerWarning(firstVersion), expectedWarning);
```

Comment:
```suggestion
    const expectedWarning = {
      skind: "invalid-semver",
      file: "package.json",
      value: "0.0.0",
      location: null,
      i18n: "sast_warnings.invalidSemVer",
      severity: "Information",
      experimental: false
    };

    assert.deepEqual(getSemVerWarning("0.0.0"), expectedWarning);
```

Refering to my previous comment:
> I'm not sure theses constants are really necessary tbh

But it's up yo you, not a necessary change ;)